README.md

```diff
@@ -1,19 +1,71 @@
-# DinguxCommander
+OD Commander is forked from DinguxCommander which is a file manager for RetroFW.
+It uses two vertical panels side by side, one being the source and the other the destination, like many 'commander-style' file managers such as Norton Commander or Midnight Commander.
 
-DinguxCommander is a file manager for RetroFW.
-It uses two vertical panels side by side, one being the source and the other the
-destination, like many 'commander-style' file managers such as Norton Commander
-or Midnight Commander.
 DinguxCommander allows to:
 
 * Copy, move and delete multiple files.
 * View a file
 * Execute a file
+* Edit a file one line at the time
 * Rename a file or directory
 * Create a new directory
 * Display disk space used by a list of selected files/dirs
 * Display disk information (used, available, total)
 
+
+## Controls
+
+* D-pad<br />
+  Move cursor/highlight<br />
+  <br />
+* L/R<br />
+  Page up/page down<br />
+  <br />
+* A<br />
+  On a directory: Open<br />
+  On a file: View or execute<br />
+  <br />
+* B<br />
+  Go to parent directory or cancel<br />
+  <br />
+* X<br />
+  Opens a menu with actions on selected item(s):<br />
```

Comment:
There is no need for `<br>` after list items

In general, in markdown, rather than using `<br>`, please use backslash at the end of the line: https://commonmark.org/help/tutorial/03-paragraphs.html

Reply:
Perfect! - I'll replace <br /> with \ throughout.

The linebreaks are there to avoid two seperate lines being merged.